### PR TITLE
backslash for tensors

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

**Comment by fredrikekre** [comment]
Is windows nightlies behind? Or why does that still pass?

**Comment by KristofferC** [comment]
Yes, they are not updating after the doc PR got merged due to an old issue.

File: docs/src/man/other_operators.md [hunk 1]
```diff
@@ -64,6 +64,16 @@ where ``\mathbf{I}`` is the second order identitiy tensor.
 inv
 ```
 
+## Backslash
+
+Left division operator such that ``\mathbf{A} \backslash \mathbf{b} = \mathbf{A}^{-1} \cdot \mathbf{b}``.
```
Comment (fredrikekre) [inline docs/src/man/other_operators.md]: if we support `Tensor{2} \ Tensor{2}` then perhaps we should also allow `Tensor{2} \ Tensor{4}` ?

**Comment by fredrikekre** [comment]
Perhaps it is just better not to support this? Its just as easy, and maybe even clearer, to write `inv(A) ⋅ b`

**Comment by KristofferC** [comment]
I think so too.

**Comment by fredrikekre** [comment]
Lets do it like this or perhaps just close this PR.

At least this provents this:

```jl
julia> A = rand(Tensor{2,3});

julia> b = rand(Vec{3});

julia> A\b
3-element Array{Float64,1}:
  0.235989
 -0.106248
  0.114865

```

**Comment by KristofferC** [comment]
lgtm

**Pull request merged**